llvm/lib/Target/RISCV/RISCVInstrInfo.td

```diff
@@ -1311,6 +1311,22 @@ def : Pat<(FrameAddrRegImm (iPTR GPR:$rs1), simm12:$imm12),
 
 /// HI and ADD_LO address nodes.
 
+let Size = 8, isReMaterializable = 1 in
+def PseudoLIaddr : Pseudo<(outs GPR:$dst), (ins uimm20_lui:$hi, simm12:$lo), []>,
```

Comment:
What about `PseudoLIAddr` or `PseudoLoadAddr`?

Reply:
Maybe PseudoLUIADDI?  I agree that LIaddr isn't super clear.

You should also add a comment above the pseudo which describes its purpose.